tests/unit/routes/test_use_case.py

```diff
@@ -339,3 +339,21 @@ async def test_delete_associated_observation_table(
             f"Cannot delete Observation Table {str(new_ob_table_id)} because it is referenced"
             in response.json()["detail"]
         )
+
+    @pytest.mark.asyncio
+    async def test_delete_use_case(self, test_api_client_persistent, create_success_response):
+        """Test delete observation_table (fail) that is already associated with a use case"""
```

Comment:
does this comment mean that we're checking whether delete observation_table works? i might be reading it wrongly, but it looks like the test seems to just be testing whether we're deleting a use case 🤔

Reply:
Yes. Because the UseCase api object also extends DeletableApiObject, so it needs to have a delete endpoint